.travis.yml

```diff
@@ -1,12 +1,22 @@
 language: java
-sudo: false
+sudo: required
```

Comment:
So this means we're back at the old and non-dockerized Travis environment, does that have any long term consequences?